tests/functional/search-auth.js

```diff
@@ -198,6 +198,48 @@ registerSuite("Search (auth)", {
         .end();
     },
 
+    "Label have correct URL after search without results"() {
+      // load up a garbage search, so we can more easily detect when
+      // the search values we want are loaded.
+      return FunctionalHelpers.openPage(
+        this,
+        url(
+          "/issues",
+          "?page=1&per_page=50&state=open&stage=all&sort=created&direction=desc"
```

Comment:
These are the default issues page params, right?

https://github.com/webcompat/webcompat.com/blob/61f4ef1be90d7d90eaec1af7a2fb75b1f055aeb9/webcompat/static/js/lib/models/issue.js#L123-L129

Probably not needed for `openPage` here if so.

Reply:
You're right. This is probably a left over of the hunt for the issue is had with the test before. Will remove.